status_deprecated.go

```diff
@@ -12,15 +11,15 @@ import (
 // NOTE: This file is only used for the ControlServer for now.
 // TODO: eventually merge this into the ControlServer and use the Progress struct.
 
-type TableStatusDeprecated struct {
+type TableStatus struct {
 	TableName                   string
 	PaginationKeyName           string
 	Status                      string
 	LastSuccessfulPaginationKey uint64
 	TargetPaginationKey         uint64
 }
 
-type StatusDeprecated struct {
+type Status struct {
```

Comment:
Can we call this something other than status, since it is a synonym for Progress? Since this struct is only used for the control server's index variable templating, it should be located in that file, instead of being on its own. The name should be changed accordingly to indicate that.

Reply:
Also, can't the Progress be embedded here to make this struct simpler? That would get rid of a bunch of copy of member variables, no?